### ISSUE TITLE
In step 4. Naming of the JSON properties could be explained

### ISSUE THREAD

**Issue by pluppplupp** [Starcounter/InvoiceDemo#42]
What's up with the $ after some of the properties?

![2015-12-14_10-55-30](https://cloud.githubusercontent.com/assets/1765023/11777865/3674ead0-a251-11e5-8235-f44b7d7257a6.png)

**Comment by Nomans** [comment]
Properties ending with $ allows user input. This is to avoid accidental write exposure.

**Comment by pluppplupp** [comment]
Ok, great. That should maybe be explained in the tutorial?!

**Comment by per-samuelsson** [comment]
We need to better document this.

Some links:

https://github.com/Starcounter/Starcounter/issues/3086
http://starcounter.io/guides/json/json-data-binding/
https://github.com/Starcounter/starcounter.io/issues/88

> That should maybe be explained in the tutorial?!

Agree. So let's keep this issue open for that.

**Comment by Nomans** [comment]
It should be explained in the tutorial the first time it is being used.

**Comment by warpech** [comment]
Thanks for reporting!

This page http://starcounter.io/guides/web/server-side-view-models/ was actually explaining `Handle (Input)` but provided a wrong JSON-by-example, where none of the properties ended with `$`. I have fixed that.

I have also added this information to the page http://starcounter.io/tutorials/1-introduction-to-part-1/1-4-create-the-view-model/

I think this issue can be closed now. Starcounter/Starcounter#3086 and Starcounter/starcounter.io#88 track documenting this even better
